src/encoding.ts

```diff
@@ -98,15 +98,15 @@ export interface Encoding extends UnitEncoding {
 export function has(encoding: Encoding, channel: Channel): boolean {
   const channelEncoding = encoding && encoding[channel];
   return channelEncoding && (
-    channelEncoding.field !== undefined ||
+    'field' in channelEncoding ||
```

Comment:
Error TS2339: Property 'field' does not exist on type 'FieldDef | PositionChannelDef | ChannelDefWithLegend | OrderChannelDef | FieldDef[] | OrderChanne...'.
  Property 'field' does not exist on type 'FieldDef[]'.